app/views/statistics/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1 class="statistic">STATISTICS</h1><br>
+  </div>
+</div>
+<% @players.each do |player| %>
+  <% if player.quarterback? %>
+     <%= render partial: 'quarterback', locals: { player: player } %>
+  <% end %>
+
+  <% if player.running_back? %>
+     <%= render partial: 'running_back', locals: { player: player } %>
+  <% end %>
+
+  <% if player.wide_receiver? %>
+     <%= render partial: 'wide_receiver', locals: { player: player } %>
+  <% end %>
+
+  <% if player.tight_end? %>
+     <%= render partial: 'tight_end', locals: { player: player } %>
+  <% end %>
+<% end %>
```

Comment:
For lines 7 - 21 see https://github.com/samehta/fantasy-metrix/pull/54/files#r52008309